Add docs for bluecurrent integration (#25014)
home-assistant/home-assistant.github.io: source/_integrations/bluecurrent.markdown

---
title: Blue Current
description: Instructions on how to integrate Blue Current chargepoints within Home Assistant.
ha_category:
  - Sensor
  - Car
ha_release: 2022.12
ha_iot_class: Cloud Push
ha_config_flow: true
ha_codeowners:
  - '@Floris272'
  - '@gleeuwen'
ha_domain: bluecurrent
ha_platforms:
  - sensor
---

[BlueCurrent](https://www.bluecurrent.nl/) is an Dutch company that makes electric car chargers.

The Blue Current integration allows you to connect to your blue current account to Home Assistant and monitor your chargepoint(s).

## Prerequisites

> 1. Login to the [Blue Current driver portal](https://bo.bluecurrent.nl/)

Reviewer: ```suggestion
1. Log in to the [Blue Current driver portal](https://bo.bluecurrent.nl/).
```

> 2. goto settings and enable developer mode

Reviewer: ```suggestion
2. Goto settings and enable developer mode.
```

> 3. Generate an api token

Reviewer: ```suggestion
3. Generate an API token.
```

{% include integrations/config_flow.md %}

## Sensor

The Blue Current integration provides the following sensors:

> <br>
> <br>

Reviewer: ```suggestion
```

Author: Removed <br> tags

> ### chargepoint sensors

Reviewer: ```suggestion
### Chargepoint sensors
```

Author: Changed to `Charge point`.

> ---
> 

Reviewer: ```suggestion
```

- average current
- average voltage
- total kW (estimate)
- energy usage in kWh
- started on
- stopped on
- offline since
- total cost in EUR
- vehicle status
- activity
- max usage in Amps

>   - the max amps the charge point can use.

Reviewer: Let's capatalize this list properly and sort it alphabetically.

Author: Changed.

> disabled by default:

Reviewer: ```suggestion
The following sensors are created as well, but disabled by default:
```

- current phase 1-3
- voltage phase 1-3
- smart charging max usage
- offline max usage

> - remaining current

Reviewer: Let's capitalize this list properly and sort it alphabetically.

> <br>

Reviewer: ```suggestion
```

> ### grid sensors

Reviewer: ```suggestion
### Grid sensors
```

> ---
> 

Reviewer: ```suggestion
```

- grid max current

> - grid average current

Reviewer: Let's capitalize this list properly and sort it alphabetically.

> disabled by default

Reviewer: ```suggestion
The following sensor is created as well, but disabled by default:
```

> - grid current phase 1-3

Reviewer: Let's capitalize this list properly.
